src/framework/preferences/js/LocalizationSchemas.js

```diff
@@ -0,0 +1,66 @@
+/*
+Copyright 2018 OCAD University
+
+Licensed under the Educational Community License (ECL), Version 2.0 or the New
+BSD license. You may not use this file except in compliance with one these
+Licenses.
+
+You may obtain a copy of the ECL 2.0 License and BSD License at
+https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
+*/
+
+var fluid_3_0_0 = fluid_3_0_0 || {};
+
+(function (fluid) {
+    "use strict";
+
+    /*******************************************************************************
+    * Starter auxiliary schema grade
+    *
+    * Contains the settings for the localization preference
+    *******************************************************************************/
+
+    // Fine-tune the starter aux schema and add localization preference
+    fluid.defaults("fluid.prefs.auxSchema.localization", {
+        gradeNames: ["fluid.prefs.auxSchema"],
+        auxiliarySchema: {
+            "terms": {
+                "templatePrefix": "../../framework/preferences/html/",
+                "messagePrefix": "../../framework/preferences/messages/"
+            },
+            "template": "%templatePrefix/SeparatedPanelPrefsEditor.html",
+            "message": "%messagePrefix/prefsEditor.json",
+            localization: {
+                "type": "fluid.prefs.localization",
+                "enactor": {
+                    "type": "fluid.prefs.enactor.localization",
+                    "classes": "@localization.classes"
```

Comment:
There is not a `localization.classes` block defined in the aux schema. Can this line be removed?

Reply:
thanks for catching that, I had made changes form some prior work and forgot to remove this.